packages/venia-concept/src/components/FilterModal/filterBlock.css

```diff
@@ -0,0 +1,59 @@
+.root:first-child {
+    border-top: 2px solid rgb(var(--venia-border));
+    border-bottom: 2px solid rgb(var(--venia-border));
+}
+
+.root {
+    border-bottom: 2px solid rgb(var(--venia-border));
+}
+
+.layout {
+    justify-content: flex-start;
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
+}
+
+.layoutGrid {
+    composes: layout;
+    display: flex;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+}
+
+.clearIcon {
+    display: flex;
+}
+
+.optionHeader {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.optionNameExpanded {
+    font-weight: 600;
+}
+
+.optionToggleButton {
+    outline: none;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    padding-top: 0.75rem;
+    padding-bottom: 0.75rem;
+}
+
+.filterList {
+    display: none;
+    padding-right: 2.75rem;
```

Comment:
IMO it would be better to not apply the padding to the swatch filter, unless of course you are going off a mockup. 

[![Image from Gyazo](https://i.gyazo.com/68cc5552282f6eade7ae0350ada9a96a.png)](https://gyazo.com/68cc5552282f6eade7ae0350ada9a96a)

Reply:
Padding is set in design: https://magento.invisionapp.com/share/S2Q41VCR45C#/screens/341558811

Reply:
Hmm it just seems...off in that view. Maybe it's better on different widths. Oh well, I'm not too concerned with this, just wanted to bring it up.

Reply:
The padding is fine. If I were to implement it, I would probably use the `grid-template-columns` property to make the accordion header and accordion container each have two columns, and just have the container's second column be empty, but it's not super important. Layout is achieved here.